core/tests/release_sources/valid_release_mail_message.txt

```diff
@@ -0,0 +1,12 @@
+Hi all,
+
+The main changes in this release are release testing.
+
+Please welcome Alice for whom this release marks their first contribution to Oppia!
+
+Thanks to Bob and Paul, our returning contributors who made this release possible.
+
+Finally, I\d like to thank Smith for their help with pre-release testing, bug-fixing and QA, as well as Tom for leading the QA team for this release.
```

Comment:
Shouldn't the backslash be an apostrophe?

Reply:
Done.